content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -0,0 +1,191 @@
+form. To manually pass or fail the check, click :guilabel:`Pass` or :guilabel:`Fail` at the top-left
+corner of the check.
+
+Alternatively, if the quality check is assigned to a |QCP| for which *norm* and *tolerance* values
+have been specified, click :guilabel:`Measure` at the top-left corner of the check instead. Doing so
+automatically marks the check as *Passed* if the recorded value is within the specified *tolerance*,
+or *Failed* if the value is outside of it.
+
+On an order
+-----------
+
+To process a *Measure* quality check on an order, select a manufacturing order or inventory order
+(receipt, delivery, return, etc.), for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+at the top of the page. Click the button to open the :guilabel:`Quality Check` pop-up window, which
+shows all of the quality checks required for that order.
+
+To process a *Measure* quality check, measure the product as instructed, then enter the value in the
+:guilabel:`Measure` field on the pop-up window. Finally, click :guilabel:`Validate` to register the
+recorded value.
+
+.. image:: measure_check/measure-check-pop-up.png
+   :align: center
+   :alt: A Measure quality check pop-up window on a manufacturing or inventory order.
+
+If the value entered is within the range specified in the :guilabel:`Tolerance` section of the
+|QCP|, the quality check passes and the pop-up window closes. The rest of the manufacturing or
+inventory order can then be processed as usual.
+
+However, if the value entered is outside of the specified range, a new pop-up window appears, titled
+:guilabel:`Quality Check Failed`. The body of the pop-up shows a warning message that states, "You
+measured # units and it should be between # units and # units.", as well as the instructions entered
+in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the pop-up, two buttons
+appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
+
+.. image:: measure_check/measure-check-failed.png
+   :align: center
+   :alt: The "Quality Check Failed" pop-up window.
+
+If the measurement was not entered correctly and should be changed, select :guilabel:`Correct
+Measure`. Doing so re-opens the :guilabel:`Quality Check` pop-up window. Enter the corrected
+measurement in the :guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check.
+
+If the measurement is correct, click :guilabel:`Confirm Measure` instead, and the quality check
+fails. Follow any instructions that were listed on the :guilabel:`Quality Check Failed` pop-up
+window.
```

Comment:
Is this the correct phrasing here? If the measurement is correct, and confirm measure is clicked, is there a reason the quality check would fail?

Just thought this was worth pointing out for a double-check to ensure the concept was conveyed in the correct way. 👍